modules/internal/SharedObject.chpl

```diff
@@ -475,6 +448,7 @@ module SharedObject {
        Equivalent to ``shared.retain(nil)``.
      */
     pragma "leaves this nil"
+    @deprecated(notes="shared.clear is deprecated - please assign shared object to `nil` instead")
```

Comment:
Please replace with "please assign ‘nil’ to the shared object".

Please make the same change for owned.clear().